ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/templates/Template4.kt

```diff
@@ -141,26 +151,44 @@ private fun Template4MainContent(
             )
         }
 
-        Packages(state, viewModel)
+        var packageSelectorVisible by remember {
+            mutableStateOf(state.templateConfiguration.mode != PaywallMode.FOOTER_CONDENSED)
+        }
 
-        ConsistentPackageContentView(state = state) {
-            IntroEligibilityStateView(
-                textWithNoIntroOffer = it.localization.offerDetails,
-                textWithIntroOffer = it.localization.offerDetailsWithIntroOffer,
-                textWithMultipleIntroOffers = it.localization.offerDetailsWithMultipleIntroOffers,
-                eligibility = it.introEligibility,
-                color = colors.text1,
-                style = MaterialTheme.typography.bodyMedium,
-                textAlign = TextAlign.Center,
-                modifier = Modifier.fillMaxWidth(),
-            )
+        Column(horizontalAlignment = Alignment.CenterHorizontally) {
```

Comment:
I needed to wrap these into another column like this to avoid the outer column `verticalArrangement` spacing. Seems like the `AnimatedVisibility` jumps when trying to animate an item disappearing and that spacing going away. I'm mimicking that by manually handling the animation of a custom `Spacer` here.

Reply:
> Seems like the AnimatedVisibility jumps when trying to animate an item disappearing and that spacing going away

Yeah that's why we can't use it in many place where e need spacing to remain consistent.